persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraPersistentEntityRegistry.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence.cassandra
+
+import java.util.{ Optional, UUID }
+
+import akka.NotUsed
+import akka.japi.Pair
+import akka.stream.javadsl
+import com.lightbend.lagom.javadsl.persistence.{ AggregateEvent, AggregateEventTag, PersistentEntityRegistry }
+
+/**
+ * Functionality specific to the Cassandra persistent entity registry.
+ */
+trait CassandraPersistentEntityRegistry extends PersistentEntityRegistry {
+
+  /**
+   * A stream of the persistent events that have the given `aggregateTag`, e.g.
+   * all persistent events of all `Order` entities.
+   */
+  def uuidEventStream[Event <: AggregateEvent[Event]](
+    fromOffset:   Optional[UUID],
```

Comment:
I like it 👍

Maybe an alternative is to not be too religious about guaranteeing binary compatible releases, since we don't yet have massive adoption and having a clean, easy-to-understand API is more important now. Also, I don't think we have set up MiMa for Lagom, which I think is another hint at the fact that binary compatibility is not our number one concern just yet.